src/components/shared/header/topBar/accountMenu/userAccount.js

```diff
@@ -66,127 +101,86 @@ class UserAccount extends React.Component {
-              >
-                <AccountInfo
-                  account={account.info[tokenKey]}
-                  token={tokenKey}
-                  t={t}
-                />
-                {
-                  tokenKey === token.active
-                    ? <span className={styles.activeLabel}>{t('Active')}</span>
-                    : null
-                }
-              </span>,
-              <Dropdown.Separator key={`separator-${tokenKey}`} className={styles.separator} />,
-            ])
-            : null
-          ))
+          this.renderTokens()
         }
-
-        <a
-          className={styles.dropdownOption}
+        <MenuItem
+          name="discord"
+          title={t('Discord')}
           href={externalLinks.discord}
-          rel="noopener noreferrer"
-          target="_blank"
           onClick={this.toggleDropdown}
-        >
-          <Icon name="discordIcon" className={styles.defaultIcon} />
-          <Icon name="discordIconActive" className={styles.activeIcon} />
-          <span>{t('Discord')}</span>
-        </a>
-
-        <a
-          className={styles.dropdownOption}
+        />
+        <MenuItem
+          name="feedback"
+          title={t('Give Feedback')}
           href={feedbackLinks.general}
-          rel="noopener noreferrer"
-          target="_blank"
           onClick={this.toggleDropdown}
-        >
-          <Icon name="feedback" className={styles.defaultIcon} />
-          <Icon name="feedbackActive" className={styles.activeIcon} />
-          <span>{t('Give Feedback')}</span>
-        </a>
-
-        <Link
-          id="settings"
+        />
+        <MenuItem
+          name="settings"
+          title={t('Settings')}
           to={routes.settings.path}
-          className={styles.dropdownOption}
           onClick={this.toggleDropdown}
-        >
-          <Icon name="settings" className={styles.defaultIcon} />
-          <Icon name="settingsActive" className={styles.activeIcon} />
-          <span>{t('Settings')}</span>
-        </Link>
-
+        />
         {
-          (typeof account.loginType === 'number' && account.loginType === loginType.normal)
+          (typeof account.loginType === 'number'
+          && account.loginType === loginType.normal
+          && token.active !== 'BTC'
+          )
             ? (
-              <Link
-                id="signMessage"
+              <MenuItem
+                name="signMessage"
+                title={t('Sign Message')}
                 to={routes.signMessage.path}
-                className={styles.dropdownOption}
                 onClick={this.toggleDropdown}
-              >
-                <Icon name="sign" className={styles.defaultIcon} />
-                <Icon name="signActive" className={styles.activeIcon} />
-                <span>{t('Sign Message')}</span>
-              </Link>
+              />
+            ) : null
+        }
+        {
+          token.active !== 'BTC'
```

Comment:
It is already defined that this page is forbidden for BTC in https://github.com/LiskHQ/lisk-desktop/blob/e7fefb9dc3bd310617a339a69200c264b946db7e/src/constants/routes.js#L108

💡 IMO it would be DRYer re-use `forbiddenTokens` here:

```suggestion
          !routes.verifyMMessage.forbiddenTokens.includes(token.active)
```

Reply:
Thanks Vit. This looks pretty verbose and less readable. plus, you have to open another file to see where is this coming from and such. I rather keep my own if you mind.

Reply:
Sure 👍